src/controllers/CheckYourAppealController.ts

```diff
@@ -46,6 +46,33 @@ export class CheckYourAppealController extends BaseAsyncHttpController {
             .chain(data => Maybe.fromNullable(data[APPEALS_KEY]))
             .extract() as Appeal;
 
+        const regionPrefix: string = appealsData.penaltyIdentifier.companyNumber.slice(0,2);
+        let internalTeam = process.env.DEFAULT_TEAM_EMAIL;
+        if (regionPrefix === 'SC') internalTeam = process.env.SC_TEAM_EMAIL;
+        else if(regionPrefix === 'NI') internalTeam = process.env.NI_TEAM_EMAIL;
+
+        // Send submission emails to internal team according to the prefix
```

Comment:
Extract to private methods instead of adding comments. Behaviour can be easily described in function name.